### DISCUSSION TITLE
List of discovered credentials, tokens, ...

### DISCUSSION THREAD

**Discussion by gano3s** [reconmap/reconmap#97] [Ideas]
We should be able to list all compromised accounts, tokens, ... in the report. So, the client can reset them once the pentest if finished.

![image](https://user-images.githubusercontent.com/2551605/155316934-80e1fe8d-3945-4797-b650-5abc70676158.png)

I would add a new tab which will contain table with all discovered/compromised/captured credentials, tokens. The UI would be similar to contacts, but with different values :) 

<img width="1785" alt="image" src="https://user-images.githubusercontent.com/2551605/155317323-31dd8931-a319-416e-ada6-9f8b465660bc.png">

**Comment by gano3s** [comment]
I created an issue for this topic: https://github.com/reconmap/reconmap/issues/98

**Comment by santiagolizardo** [comment]
Closing discussion as the feature has been implemented. Thanks
